README.md

```diff
@@ -1,8 +1,20 @@
 # image-distribution-operator
-// TODO(user): Add simple overview of use/purpose
+
+This operator runs on a kubernetes management cluster.
+Its purpose is to propagate node os images that are needed to create workload clusters to
+the workload clusters respective image catalog.
 
 ## Description
-// TODO(user): An in-depth paragraph about your project and overview of use
+
+The image-distribution-operator is made up of two controllers:
+
+### `release-controller`
+The `release-controller` watches release custom resources on the management cluster.
+It will generate the os image name from each release and keep track of the images that are needed to create workload clusters.
+The release-controller syncs the image names to a ConfigMap on the management cluster.
```

Comment:
maybe not accurate anymore after the discussion today but it can always be changed later :)

Reply:
Yeah :grimacing: I'm working on another PR to update these things